app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -231,5 +231,5 @@
 </body>
 
 <!-- always insert JS here -->
-<%= inline_js('user_analytics.js') %>
+<%= inline_js("standalone/user_analytics.js") %>
```

Comment:
Looks like we have to update the Help path too.

Reply:
What a darn great spec that caught it.

Reply:
MAKE SURE IT RENDERS PLEASE OKTHANKSBYE.